pokemon-showdown

```diff
@@ -19,7 +19,7 @@ try {
 	require.resolve('sockjs');
 } catch (e) {
 	console.log('Installing dependencies...');
-	child_process.execSync('npm install --production', {stdio: 'inherit'});
+	child_process.execSync('npm install --production --no-optional', {stdio: 'inherit'});
```

Comment:
`sockjs` is required

`node-static` and `cloud-env` are optional but should be installed by default

`ofe` and `nodemailer` are optional and should not be installed by default